addons/sales_team/migrations/8.0.1.0/post-migration.py

```diff
@@ -20,12 +20,19 @@
 #
 ##############################################################################
 
+import logging
 from openerp import pooler, SUPERUSER_ID
 from openerp.openupgrade import openupgrade, openupgrade_80
 
+logger = logging.getLogger('OpenUpgrade.sales_team')
+
 
 @openupgrade.migrate(no_version=True)
```

Comment:
Hi @pedrobaeza. 
Thanks a lot for your review.

- sales_team doesn't exist in V7.0.
- this module will be installed on V8.0, by dependencies.
- this migration script should be executed if crm is installed on V7.0.
- this migration script should not be executed if crm is not installed on V7.0.

so no_version should stay, but crm test is necessary.

What do you think ?